appointment-svc/service/db-service/AppointmentRepository.ts

```diff
@@ -153,9 +153,25 @@ class AppointmentRepository {
 
     if (!availableAppointment) {
       console.log('No available appointment at this time');
+    } else {
+      return availableAppointment;
     }
+  }
+
+  async getOpenAppointments(currentDate: Date) {
+    const targetDate = new Date(currentDate);
+    targetDate.setDate(currentDate.getDate() + 1);
```

Comment:
You could add second date parameter to make this endpoint more flexible instead of hardcoding it

Reply:
Also, you could rename it instead of 'open' and name it 'canceled' 
It's just a preference because we don't have value open, so up to you

Reply:
the reason i only kept it one date was because our service will only be calling the open appointments from one day after the currentDate. i can add in a second parameter for date when we decide to level up our project but im fine with just leaving it hardcode to + 1 day. 

yeah i can change to canceled instead of open